Week-1/README.md

```diff
@@ -0,0 +1,46 @@
+Assignment 1's question 2 file name is Assignment1_question2
+To run the file from terminal Assignment1_question2.html and execute it
+
+Assignment 1's question 1 is mentioned below:
+
+Web Browser
+It is tool used to access the internet.Web browser allow you to view websites.
+E.g.
+Chrome ,safari,
+
+Functions of browsers
+->Retrieve information from Word wide web and make it availble to users.
+->Can be used to visit websites.
+->Can open multiple web pages at the same time
+
+Components of Browser.
+->User Interface.
+->Browser Engine
+->Rendering Engine
+->Networking
+->Javascript Interpreter
+->Ui Backend
+
+User Interface-
+Everything you can see but you can't manipulate.Address Bar, settings of your browser.
+
+Browser Engine-
+Works as a mediator between user interfae and rendering engine.If you press refresh then browser engine will refresh that page for you.
+
+Rendering engine-
+Parses the html ,css and javascript
+When you open the website rendering engine has been started.
+
+Parsing-
+It is of two types conventional and unconventional parser.
+Parser
+
+Parsing can be defined as translating a document into a structure that code can use.
+Parsing consist of grammar and syntax rules.
+
+Networking-
+When you have https request it makes sure network resources are loaded.
+
+Rendering engine –
+It’s a software that draws text and images on screen.The engine draws structured text from a Html document
+And format it properly based on given style.
```

Comment:
Yes I will add the images , and the points remaining ,Just one question  why README.md file is chosen for such type of questions why not Word file ?

Reply:
docx file cannot be opened unless you have MS Word installed. Also, readme.md is directly supported by git and can be viewed online directly instead of having to download a file.